apm-server/examples/default/test/goss.yaml

```diff
@@ -1,6 +1,7 @@
 http:
-  http://localhost:8200?pretty:
+  https://localhost:8200?pretty:
```

Comment:
This PR is only updating ES to use security, so only the query to ES (`http://XXX:9200`) should be updated to https. The query to the others services like apm here should stay in http for now.